core/domain/blog_domain_test.py

```diff
@@ -588,3 +607,11 @@ def test_blog_post_tags_passes_validation(self) -> None:
         self._assert_valid_tag_elements(
             'Expected each tag in \'tags\' to be a string, received: '
             '\'123\'')
+
+    def test_blog_post_summary_dict_content(self) -> None:
+        """Test that to_dict preserve all data within a
+        blog post summary contents object.
+        """
+        blog_post_summary_contents_dict = self.blog_post_summary.to_dict()
+        self.assertEqual(
+            self.blog_post_summary.to_dict(), blog_post_summary_contents_dict)
```

Comment:
Doesn't this just check that `self.blog_post_summary.to_dict() == self.blog_post_summary.to_dict()`? That doesn't seem like a check that would ever fail

Reply:
@div-yam A better way might be to compare the output of `to_dict` to a sample hard-coded dictionary.